pages/common/openvpn.md

```diff
@@ -0,0 +1,16 @@
+# openvpn
+
+> OpenVPN client and daemon binary.
+> More information: <https://openvpn.net/>.
+
+- Connect to server with config file:
+
+`sudo openvpn {{client.conf}}`
+
+- Try to set up an insecure peer-to-peer tunnel on bob.example.com host:
```

Comment:
Hrm, if this is insecure I'm reluctant to include it. We should always promote good security practices where possible.

Reply:
Well, It is actually based on man page examples. I'll add simple secured version. Therefore I see no reason to eliminate this, because it described as insecure, it is simple and it gives a point to play with. And it is secure in a trusted network.

Reply:
Ah, I see. Including the secure version definitely helps.